### PR TITLE
Moving build to dotnet

### LINKED ISSUE #149: Change to build to use dotnet

### PR DESCRIPTION
**Pull request by moozzyk**
Fixes #149

### CODE REVIEW

**Comment by victorhurdugaci** [comment]
If the makefile doesn't have anything special in it, you can remove it completely

**Comment by moozzyk** [comment]
Yeah. It's pretty much empty. Will remove then and see if it still builds.

**Comment by moozzyk** [comment]
:up::calendar:

- removed makefile
- updated .travis.yml

**Comment by victorhurdugaci** [comment]
:shipit: once the tests pass on travis

File: .travis.yml [hunk 1]
```diff
@@ -10,9 +10,11 @@ addons:
...
-  - KOREBUILD_DNU_RESTORE_CORECLR=true
 mono:
   - 4.0.5
+os:
+  - linux
+  - osx
+osx_image: xcode7.1
 script:
-  - ./build.sh --quiet verify
+  - ./build.sh verify
```
Comment (moozzyk) [inline .travis.yml]: @victorhurdugaci - this causes build failures - it wants me to specify `-m`. I thought you said that it is no longer required to have a makefile if it does not have any customizations... Also, I see that in other repos -m is not specified either and apparently  build runs. What am I missing?

Comment (victorhurdugaci) [inline .travis.yml]: Make sure you get the latest `build.sh` from Universe / build-template-dotnet. We've updated the templates yesterday

Comment (moozzyk) [inline .travis.yml]: Thanks! That must have been it. I copied the latest templates and it builds now.

**Comment by davidfowl** [comment]
:shipit:

**Pull request closed without merging**